src/style.css

```
@@ -0,0 +1,85 @@
+body {
+  margin: 0;
+ 
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  font-family: Arial, Helvetica, sans-serif;
+  color: #fff;
+  font-size: 100px;
+}
+
+.block {
+  display: flex;
+  flex-basis: 100%;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+}
+
+.block--1 {
+  background-color: #f00;
+}
+
+.block--2 {
+  background-color: #c00;
+}
+
+.block--3 {
+  background-color: #900;
+}
+
+.block--4 {
+  background-color: #600;
+}
+
+.block--5 {
+  background-color: #300;
+}
+
+.block--6 {
+  background-color: #000;
+}
+
+@media (min-width: 600px) and (max-width: 899px) {
```

```suggestion
@media (min-width: 600px) {
```
Следующий медиа запрос перекроет этот, так что and можно не писать